src/omg-cli/omg/setup/ops_manager.go

```diff
@@ -70,8 +70,19 @@ func (s *OpsManager) Unlock() error {
 	}
 }
 
-func (s *OpsManager) ApplyChanges() error {
-	return s.om.ApplyChanges()
+func (s *OpsManager) ApplyChangesPAS() error {
```

Comment:
Can you add GoDoc to these methods?

I know the original one didn't have godoc, but we should work on adding them as we go.

Reply:
I changed my mind. I created an issue for adding docstrings throughout the project. (see #43) We can do that later.